src/website/views/components/spinner/_examples.pug

```diff
@@ -132,34 +132,38 @@ h4 Spinners in Text Inputs
 .example.-mb--3
   .-p--3
     .chi-form__item.-mb--2(style="max-width:20rem")
-      label.chi-label(for="spinner-input-01") Label
-      .chi-input__wrapper
-        input(type="text", class="chi-input", placeholder="Placeholder", value="Sample Text" id="spinner-input-01")
-        svg(class=`chi-spinner -icon--muted`,viewBox=`0 0 66 66`)
-          title Loading
-          circle(class=`path`,cx=`33`,cy=`33`,r=`30`,fill=`none`,stroke-width=`6`)
+      chi-label(for="spinner-input-01") Label
```

Comment:
I would recommend adding this example to the Text input component page too to ensure developers don't miss it. We could title it "Spinner" and place it under the Placeholder example.

Reply:
Something along the lines of this for the text input page:
```
h3 Spinner
p.-text
  | Use the spinner attribute to render a spinner inside a text input. This lets users know when information is saving or loading.
```

Reply:
Good point